src/ConnectedMode/ConnectedModePackage.cs

```diff
@@ -75,6 +75,9 @@ protected override async Task InitializeAsync(CancellationToken cancellationToke
 
             hotspotDocumentClosedHandler = componentModel.GetService<IHotspotDocumentClosedHandler>();
 
+            hotspotStoreMonitor = componentModel.GetService<ILocalHotspotStoreMonitor>();
+            await hotspotStoreMonitor.InitializeAsync();
```

Comment:
Create the monitor in the ConnectedMode package, same as for the hotspot document handler.